master/internal/resourcemanagers/resource_pool.go

```diff
@@ -366,7 +366,7 @@ func (c containerAllocation) Summary() sproto.ContainerSummary {
 }
 
 // StartContainer notifies the agent to start a container.
-func (c containerAllocation) Start(ctx *actor.Context, spec image.TaskSpec) {
+func (c containerAllocation) Start(ctx *actor.Context, spec image.TaskSpec, rank int) {
```

Comment:
side note, why is tasks imported as image? doesn't really make sense to me

Reply:
I don't know why it is imported as `image`. I just didn't change it. Probably we should change ti